thrift/src/main/thrift/atoms/timeline.thrift

```diff
@@ -0,0 +1,19 @@
+namespace * contentatom.timeline
+namespace java com.gu.contentatom.thrift.atom.timeline
+#@namespace scala com.gu.contentatom.thrift.atom.timeline
+
+include "entity.thrift"
+include "../shared.thrift"
+
+struct Timeline {
+  1: optional string typeLabel
+  2: optional list<string> tags
+  3: required list<TimelineItem> events
+  4: optional Entity entity
+}
+
+struct TimelineItem {
+  1: required string title
```

Comment:
I think there could be more than one, also could events be associated with a timeline? (I'm not sure how entities, events and stories relate at present..)